signup/signup.js

```diff
@@ -0,0 +1,66 @@
+import {
+  emailInput,
+  pwInput,
+  pwCheckInput,
+  eyeBtn,
+  pwCheckEyeBtn,
+  signupBtn,
+} from "../utils/tags.js";
+import {
+  emailDuplicatedMsg,
+  handleEmailInputEmptyValueCheck,
+  handleEmailInputInvalidValueCheck,
+  handleInputDuplicatedCheck,
+  emailEmptyMsg,
+  emailInvalidMsg,
+  emailRegex,
+  handlePwInputDoubleCheck,
+  handlePwInputInvalidCheck,
+  pwRegex,
+  pwInvalidMsg,
+  handleEyeBtnClick,
+  pwNotMachedMsg,
+} from "../utils/auth.js";
+
+function handleSignup() {
+  handleEmailInputEmptyValueCheck(emailInput, emailEmptyMsg);
+  handleEmailInputInvalidValueCheck(emailRegex, emailInput, emailInvalidMsg);
+  handlePwInputDoubleCheck(pwInput, pwCheckInput, pwNotMachedMsg);
+  handlePwInputInvalidCheck(pwRegex, pwInput, pwInvalidMsg);
+  if (
+    !emailInput.querySelector("input-error") &&
+    !pwInput.querySelector("input-error")
+  ) {
+    location.href = "/folder";
+  }
+}
+
+function handleSignupEnter(e) {
+  if (e.key === "Enter") {
+    handleSignup();
+  }
+}
+
+emailInput.addEventListener("blur", () =>
+  handleEmailInputEmptyValueCheck(emailInput, emailEmptyMsg)
+);
```

Comment:
```suggestion
const handleEmailBlur = () => {
  if (!isEmailValid()) addErrorMessage(emailInput, emailEmptyMsg)
}

emailInput.addEventListener("blur", handleEmailBlur);
```
핸들러를 조금 더 유지보수 하기 좋은 형태로 구현해본다면 pseudo code로는 대략 이렇게 되지 않을까 싶어요! blur가 될 때 유효성 검사를 하고 그 후 에러 메시지 노드를 삽입한다는 건 페이지 단위에서 적용되는 비즈니스 로직이기 때문에 여기서 조립을 하는 느낌으로요! 
그리고 이렇게 익명이 아닌 핸들러를 정의해두면 추후 removeEventListener 할 일이 생길 때도 편합니다! ㅎㅎ